com.unity.render-pipelines.universal/Runtime/2D/Light2DBatch.cs

```diff
@@ -0,0 +1,163 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Unity.Collections;
+using Unity.Mathematics;
+using UnityEngine.Experimental.Rendering.Universal.LibTessDotNet;
+using UnityEngine.Rendering;
+using UnityEngine.U2D;
+
+namespace UnityEngine.Experimental.Rendering.Universal
+{
+
+    internal struct ShapeMeshBatch
+    {
+        internal int hashCode;
+        internal int meshCount;
+        internal int startHash;
+        internal int endHash;
+    }
+
+    internal static class Light2DBatch
```

Comment:
Please simplify this... 

Why is it static?
Why do we need a mesh pool? 
What is s_ActiveBatchHashes for?